src/bindings/haskell/CMakeLists.txt

```diff
@@ -6,8 +6,7 @@ if (CABAL_EXECUTABLE) # set by find_program
 if (C2HS_EXECUTABLE)
 if (GHC_EXECUTABLE)
 	
-	SET (CABAL_INCLUDE_DIRS "${CMAKE_SOURCE_DIR}/src/include, ${CMAKE_BINARY_DIR}/src/include")
-	SET (CABAL_EXTRA_LIB_DIRS "")
+	set (CABAL_INCLUDE_DIRS "${CMAKE_SOURCE_DIR}/src/include, ${CMAKE_BINARY_DIR}/src/include")
```

Comment:
I think 

```cmake
set (CABAL_INCLUDE_DIRS "\"${CMAKE_SOURCE_DIR}/src/include\", \"${CMAKE_BINARY_DIR}/src/include\"")
```

would be better, since it makes people that have space characters in their build path happier 😊. As far as I can tell `@CMAKE_CURRENT_SOURCE_DIR@` in `libelektra-haskell.cabal.in` should also be quoted:

```
"@CMAKE_CURRENT_SOURCE_DIR@"/src
```

.